### PR TITLE
Change function name

### PR DESCRIPTION
**Pull request by jagruti-gothwal**

### CODE REVIEW

File: public/js/notes.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 let tableBody = document.getElementById("tableBody");
 
 // push the data in localstorage
-function updateNote() {
+function fetchNote() {
```
Comment (pr656d) [inline public/js/notes.js]: Are you fetching only 1 note from local storage?

**Review by pr656d — CHANGES_REQUESTED** [review]

File: public/js/notes.js (hunk 1 above)
Comment (jagruti-gothwal) [reply public/js/notes.js]: No

File: public/js/notes.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 let tableBody = document.getElementById("tableBody");
 
 // push the data in localstorage
```
Comment (pr656d) [inline public/js/notes.js]: Are you pushing in local storage or pulling from local storage?

File: public/js/notes.js [hunk 3]
```diff
@@ -11,24 +11,24 @@ function updateNote() {
...
+            <td><button id="dBtn" onclick="deleteNotes(${index});">Delete</button></td>
             </tr>`;
     });
   } else {
     console.log("Error in adding notes");
   }
   tableBody.innerHTML = str;
 }
-updateNote(); //calling update to update table
+fetchNotes(); //calling update to update table
```
Comment (pr656d) [inline public/js/notes.js]: Comment is not necessary here. Also comment is wrong.

File: public/js/notes.js [hunk 4]
```diff
@@ -11,24 +11,24 @@ function updateNote() {
...
 
 //deletes the data from table and localstorage
-function deleteNote(itemIndex) {
+function deleteNotes(itemIndex) {
```
Comment (pr656d) [inline public/js/notes.js]: Are you deleting a note based on "itemIndex"?

Comment (pr656d) [reply public/js/notes.js]: Wrong function name, if your function is deleting 1 entry based on provided "itemIndex" then it name should be "deleteNote" and if deleting multiple based on array of indexes then it's "deleteNotes". Function name should be meaningful.

File: public/js/notes.js [hunk 5]
```diff
@@ -11,24 +11,24 @@ function updateNote() {
             <th scope="row">${index + 1}</th>
             <td>${element[0]}</td>
             <td>${element[1]}</td>
-            <td><button id="dBtn" onclick="deleteNote(${index});">Delete</button></td>
+            <td><button id="dBtn" onclick="deleteNotes(${index});">Delete</button></td>
```
Comment (pr656d) [inline public/js/notes.js]: "deleteButton" or "deleteBtn" would be good ID. Avoid using short ID name. "dBtn" can denote "destroy button", "doll button" or something else.

Comment (pr656d) [reply public/js/notes.js]: Change everywhere if you're using this types of names in code in seperate branch.

**Review by pr656d — CHANGES_REQUESTED** [review]

**Pull request merged**